app-network/lighttpd/autobuild/overrides/usr/lib/systemd/system/lighttpd.service

```
@@ -4,8 +4,10 @@ After=syslog.target network.target
 
 [Service]
 PrivateTmp=true
-ExecStart=/usr/bin/lighttpd-angel -D -f /etc/lighttpd/lighttpd.conf
-ExecReload=/usr/bin/kill -HUP $MAINPID
+ExecStartPre=/usr/sbin/lighttpd -tt -f /etc/lighttpd/lighttpd.conf
+ExecStart=/usr/sbin/lighttpd -D -f /etc/lighttpd/lighttpd.conf
```

We have `/usr/bin` and `/usr/sbin` merged in AOSC OS, so it should be like...

```suggestion
ExecStartPre=/usr/bin/lighttpd -tt -f /etc/lighttpd/lighttpd.conf
ExecStart=/usr/bin/lighttpd -D -f /etc/lighttpd/lighttpd.conf
```